docuilib/src/theme/ReactLiveScope/index.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import products from '../../assets/data/products';
-require('./configurations');
```

Comment:
See the icons here `chevronRight` and `drag` - they are probably broken now (see `SortableList`)